Rename and move web interface guide (#1382)
meilisearch/documentation: resources/open-api-readme.md

Every one of Meilisearch's [API endpoints](/reference/api) is also documented with [OpenAPI v3.1.0](http://spec.openapis.org/oas/v3.1.0).

> The raw YAML file is available [here](https://bump.sh/doc/meilisearch.yaml) and a search preview interface to see it [here](https://bump.sh/doc/meilisearch).

Reviewer: ```suggestion
The raw YAML file is available [here](https://bump.sh/doc/meilisearch.yaml) and a browser interface [here](https://bump.sh/doc/meilisearch).
```
